sample/adder/app/src/main/java/Job.java

```diff
@@ -27,6 +29,13 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 
     protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 
+        LOGGER.severe("Logging an SEVERE-level message");
+        LOGGER.info("Logging an INFO-level message");
+        LOGGER.warning("Logging an WARNING-level message");
+        LOGGER.fine("Logging an FINE-level message");
+        LOGGER.finer("Logging an FINER-level message");
+        LOGGER.finest("Logging an FINEST-level message");
```

Comment:
> it logs only WARNING, INFO and SEVERE levels.

If it doesn't work, maybe it's not a good idea to put it inside an exemplary job.

Reply:
It depends on logging config and I thought this example would show to users which levels appear by default and which don't (so they don't have to experiment themselves). But I don't have strong oppinion here, so will remove those.